### PR TITLE
Cheaper Steam Age

### PR DESCRIPTION
**Pull request by Syrcan**

### CODE REVIEW

File: scripts/steam_age.zs [hunk 1]
```diff
@@ -113,4 +148,4 @@ metal_bender.recipeBuilder().EUt(8).duration(100)
     .buildAndRegister();
 
 // Manual Copper Foil
-recipes.addShaped("gt_copper_foil", <ore:foilCopper>.firstItem * 2, [[<ore:gregHardHammers>, <ore:plateCopper>]]);
\ No newline at end of file
+recipes.addShaped("gt_copper_foil", <ore:foilCopper>.firstItem * 2, [[<ore:gregHardHammers>, <ore:plateCopper>]]);
```
Comment (TechLord22) [inline scripts/steam_age.zs]: This recipe won't be needed in the next gregicality update.

File: scripts/steam_age.zs [hunk 2]
```diff
@@ -20,16 +32,39 @@ val name_removals = [
...
 }
+for item in name_yeets {
+    mods.jei.JEI.removeAndHide(item);
+}
 
 recipes.addShaped("gt_coke_clay", <metaitem:compressed.coke.clay> * 8, [[<ore:ingotClay>, <ore:ingotClay>, <ore:ingotClay>],[<ore:sand>, <ore:formWood>, <minecraft:sand>], [<ore:sand>, <ore:sand>, <ore:sand>]]);
-recipes.addShaped("gt_bronze_hull", <gregtech:machine_casing:10>, [[null, <ore:plateBronze>, null],[<ore:plateBronze>, <ore:craftingToolHardHammer>, <ore:plateBronze>], [null, <ore:plateBronze>, null]]);
+recipes.addShaped("gt_coke_hatch", <meta_tile_entity:coke_oven_hatch>, [[null, <ore:plateIron>, null],[<ore:gregHardHammers>, <gregtech:metal_casing:8>, <ore:gregWrenches>],[null, <ore:blockGlassColorless>, null]]);
+recipes.addShaped("gt_bronze_hull", <gregtech:machine_casing:10>, [[null, <ore:plateBronze>, null],[<ore:plateBronze>, <ore:gregHardHammers>, <ore:plateBronze>], [null, <ore:plateBronze>, null]]);
 recipes.addShaped("gt_solar_boiler", <meta_tile_entity:steam_boiler_solar_bronze>, [[null, <ore:blockGlassColorless>, null],[<ore:pipeSmallBronze>, <ore:plateSilver>, <ore:pipeSmallBronze>], [null, <gregtech:machine_casing:11>, null]]);
```
Comment (TechLord22) [inline scripts/steam_age.zs:54]: Do we maybe want to change the silver to something different here?

**Review by TechLord22 — APPROVED** [review]

**Comment by TechLord22** [comment]
I'd also like to suggest faster steel recipes using wrought iron in the PBF if they do not exist already.

File: scripts/steam_age.zs (hunk 2 above)
Comment (Syrcan) [reply scripts/steam_age.zs:54]: Now that I think about it, if we gate silver behind the "Deep Dark" we definitively need another metal. No idea which

Comment (htmlcsjs) [reply scripts/steam_age.zs:54]: Bronze maybe and change textures

Comment (Syrcan) [reply scripts/steam_age.zs:54]: I think silver is fine as is. If players *really* want solars, they can buy silver dusts with coins (we can add that).
Worst case I'll do another PR in the future.

File: scripts/steam_age.zs (hunk 1 above)
Comment (Syrcan) [reply scripts/steam_age.zs]: Will fix later, once the modpack receives the updated Gregicality and the recipe will be duped.

**Pull request merged**